CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to wradlib
+
+The wradlib project encourages everyone to contribute to the developement of wradlib. This paper describes the general guidelines for contributing to wradlib. There is more information on [wradlib's community effort](http://wradlib.org/wradlib-docs/latest/community.html) at the project's main website [wradlib.org](wradlib.org).
+
+In general wradlib uses GitHub Pull Requests for contribution. Second, the [wradlib developers](CONTRIBUTORS.txt) have GitHub Commit Access to the project's main repository.
+
+## Code of Conduct
+
+Contributors to wradlib are expected to act respectfully toward others in accordance with the [OSGeo Code of Conduct](http://www.osgeo.org/code_of_conduct).
+
+## Contributions and Licensing
+
+All contributions shall comply to the project [license](LICENSE.txt).
+
+## GitHub
+
+Source code, testing, bug tracking as well as docs and wiki are handled via GitHub.
+
+## Documentation
+
+* wradlib's documentation is administered in doc/-folder, in reStructuredText format.
+* Jupyter Notebooks (in notebooks/-folder) are utilized to extend the documentation with examples and tutorials.
+* wradlib uses [Sphinx](http://sphinx-doc.org/), [runipy](https://github.com/paulgb/runipy) and [nbsphinx]()https://github.com/spatialaudio/nbsphinx to compile docs
+
+## Bugs
```

Comment:
Maybe rather use "Issue tracking" as header? - as the section is about issue tracking, not only bugs.